Toolbar1Async/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
A couple things to note:
- The `PluginEntry` defines the "entry point" of the plugin, effectively the "index.html" for any website.
- There is no `toolbar.json`. This additional file seems unnecessary to me. If it's really desirable, I can make it work for the async case.
- Currently you can't make a panel and a toolbar in a single plugin, but that is completely doable, I just didn't workout what the manifest would look like in that case.

Reply:
Looks like it already works (more or less).

Reply:
It's nice that a single plugin can be a panel and a toolbar. Nice work!